job-dsls/jobs/prod/prod_bamoe_staging.groovy

```diff
@@ -18,7 +18,7 @@ pipelineJob("${folderPath}/bamoe-staging") {
     }
 
     parameters {
-        stringParam("PNC_API_URL", "http://orch.psi.redhat.com/pnc-rest/v2", "PNC Rest API endpoint. See: https://docs.engineering.redhat.com/display/JP/User%27s+guide")
+        stringParam("PNC_API_URL", "\${env.ORCH_PSI_URL}/pnc-rest/v2", "PNC Rest API endpoint. See: \${env.DOCS_ENGINEERING_URL}/display/JP/User%27s+guide")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.